bench-tps/src/cli.rs

```diff
@@ -316,30 +316,30 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
         .arg(
             Arg::with_name("rpc_client")
                 .long("use-rpc-client")
-                .conflicts_with("tpu_client")
+                .conflicts_with("thin_client")
                 .takes_value(false)
                 .help("Submit transactions with a RpcClient")
         )
         .arg(
-            Arg::with_name("tpu_client")
-                .long("use-tpu-client")
+            Arg::with_name("thin_client")
+                .long("use-thin-client")
                 .conflicts_with("rpc_client")
                 .takes_value(false)
-                .help("Submit transactions with a TpuClient")
```

Comment:
We should preserve the `--use-tpu-client` flag (and the various `conflicts_with` logic), even though it duplicates the (new) default behavior, to avoid breaking downstream users.